service/src/main/java/tds/assessment/repositories/impl/AdminSubjectQueryRepositoryImpl.java

```diff
@@ -44,7 +45,8 @@ public AdminSubjectQueryRepositoryImpl(final DataSource dataSource) {
                     rs.getString("key"),
                     rs.getString("assessmentId"),
                     rs.getBoolean("issegmented"),
-                    rs.getString("selectionalgorithm")));
+                    rs.getString("selectionalgorithm"),
+                    rs.getFloat("startAbility")));
```

Comment:
Interesting point. Looking at production, stating, and deployment legacy itembank.tblsetofadminsubject tables, it appears that this value defaults to 0. However, taking a look at the CREATE statement for that table, it appears to be DEFAULT NULL. 

Looking a bit further into the itembank stored procedure that loads this table data (load_adminsubjects.sql in TDS_TestDeliverySystemDataAccess), it looks like the default value is set to 0. Based on this, I think we're safe to assume null isn't important and we can make the property a primitive float.